test/boost/cql_query_test.cc

```diff
@@ -2397,8 +2397,8 @@ SEASTAR_TEST_CASE(test_in_restriction) {
         {
             auto msg = e.execute_cql("select r1 from tir2 where (c1,r1) in ((0, 1),(1,2),(0,1),(1,2),(3,3)) ALLOW FILTERING;").get0();
             assert_that(msg).is_rows().with_rows({
-                {int32_type->decompose(1), int32_type->decompose(0)},
-                {int32_type->decompose(2), int32_type->decompose(1)},
+                {int32_type->decompose(1)},
+                {int32_type->decompose(2)},
```

Comment:
And why this?

Reply:
ignore

Reply:
Explained in the commit message.  The extra values were read to perform filtering, and they're visible to the test.

Reply:
This is https://github.com/scylladb/scylla/issues/4126. Interestingly, this extra values was the main reason why I lost faith in doing CQL tests via some invented interface (be it cql_tests.cc as here or cql_repl) and think we should be doing CQL tests via a real CQL driver.